src/_js/main.mjs

```diff
@@ -91,4 +104,4 @@ ga.q = [];
 ga('create', UA_ID, 'auto');
 ga('set', 'referrer', document.referrer.split('?')[0]);
 ga('send', 'pageview');
-insertScript('https://www.google-analytics.com/analytics.js');
+import('https://www.google-analytics.com/analytics.js');
```

Comment:
@tomayac @mathiasbynens Looks like no-cors error still appears on v8.dev - apparently because `import` is using CORS requests whereas regular scripts don't (TIL: https://v8.dev/features/dynamic-import).

It's a shame as `import` is kind of nice for situations like this, but I guess I'll have to change it back for Google Analytics...